lightningd/options.c

```diff
@@ -793,6 +793,11 @@ void register_opts(struct lightningd *ld)
 	opt_register_arg("--bitcoin-rpcport", opt_set_talstr, NULL,
 			 &ld->topology->bitcoind->rpcport,
 			 "bitcoind RPC port");
+	opt_register_arg("--bitcoin-try-timeout", opt_set_u64, NULL,
```

Comment:
:warning: bikeshedding here :warning: 

I'd prefer `--bitcoin-retry-timeout` since it's not a single attempt but actually counts the time between the first failure and the first success.